### ISSUE TITLE
Apt Repository

### ISSUE THREAD

**Issue by mrgou** [mb21/panwriter#35]
Is there an up-to-date apt repository for PanWriter?

**Comment by mb21** [comment]
Nope, as far as I know that's not a very common thing to do for electron applications?

**Comment by mrgou** [comment]
Sure it is. Typora is installed with apt, and for VSCode, Atom... once you install the deb package, it registers a repository and the package is kept up to date. Much more convenient than an AppImage.

**Comment by mb21** [comment]
Okay... well, let's see how many upvotes this gets...
pull request to generate the deb package and upload it to the debian repo or wherever using a github action or whatever welcome!

**Comment by alexmyczko** [comment]
challenge accepted, but someone needs to fix this first: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=842420

here's the panwriter deb packaging so far:
http://sid.ethz.ch/debian/panwriter/

**Comment by mb21** [comment]
Thanks @alexmyczko! I'd gladly accept any pull requests to make an "official apt package"...

To release a new version, currently I simply run`yarn run release` on my laptop, which uploads all build artifacts to a new release on github... but if somebody has the time, we could also set up a github action that does the build and potentially pushes a apt package somewhere...

**Comment by alexmyczko** [comment]
"a apt" sounds like stuttering and wrong, JFYI: https://people.phys.ethz.ch/~myczko/a-or-an/

**Comment by Moonbase59** [comment]
+1 for `.deb` packages and an apt repo. The Debian/Ubuntu/Mint user base is large and not everyone likes AppImages. Installation using a PPA might be easier, and who knows, maybe it’ll end up in the Debian/Ubuntu/Mint repos eventually…
